chapter_6/Readme.md

```diff
@@ -0,0 +1,18 @@
+>###Find all the places where a string is put inside a string. There are four places.
+
+```x = "There are #{types_of_people} types of people."```
+```y = "Those who know #{binary} and those who #{do_not}."```
+```joke_evaluation = "Isn't that joke so funny?! #{hilarious}"```
```

Comment:
Only need the last backticks, but make sure they are on a new line by themselves